### PR TITLE
crunchyroll-go: Add version 2.1.0

### PR DESCRIPTION
**Pull request by AdmnJ**
Added an app manifest for the crunchyroll-go CLI tool.
App can be found here https://github.com/ByteDream/crunchyroll-go

Closes #XXXX

Relates to #XXXX

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Use this:
```json
{
    "version": "2.1.0",
    "description": "A Go library & CLI for undocumented Crunchyroll APIs.",
    "homepage": "https://github.com/ByteDream/crunchyroll-go",
    "license": "LGPL-3.0-or-later",
    "suggest": {
        "ffmpeg": "ffmpeg",
        "vcredist": "extras/vcredist2010"
    },
    "notes": "You need a premium account for full (API) access.",
    "architecture": {
        "64bit": {
            "url": "https://github.com/ByteDream/crunchyroll-go/releases/download/v2.1.0/crunchy-v2.1.0_windows.exe#/crunchy.exe",
            "hash": "76b746c16269981230c68bb4f85fc0a35ceaf68a124e9f882ee3584ea179d355"
        }
    },
    "bin": "crunchy.exe",
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/ByteDream/crunchyroll-go/releases/download/v$version/crunchy-v$version_windows.exe#/crunchy.exe"
            }
        }
    }
}
```

**Comment by AdmnJ** [comment]
> Use this:
> 
> ```json
> 
> {
> 
>     "version": "2.1.0",
> 
>     "description": "A Go library & CLI for undocumented Crunchyroll APIs.",
> 
>     "homepage": "https://github.com/ByteDream/crunchyroll-go",
> 
>     "license": "LGPL-3.0-or-later",
> 
>     "suggest": {
> 
>         "ffmpeg": "ffmpeg",
> 
>         "vcredist": "extras/vcredist2010"
> 
>     },
> 
>     "notes": "You need a premium account for full (API) access.",
> 
>     "architecture": {
> 
>         "64bit": {
> 
>             "url": "https://github.com/ByteDream/crunchyroll-go/releases/download/v2.1.0/crunchy-v2.1.0_windows.exe#/crunchy.exe",
> 
>             "hash": "76b746c16269981230c68bb4f85fc0a35ceaf68a124e9f882ee3584ea179d355"
> 
>         }
> 
>     },
> 
>     "bin": "crunchy.exe",
> 
>     "checkver": "github",
> 
>     "autoupdate": {
> 
>         "architecture": {
> 
>             "64bit": {
> 
>                 "url": "https://github.com/ByteDream/crunchyroll-go/releases/download/v$version/crunchy-v$version_windows.exe#/crunchy.exe"
> 
>             }
> 
>         }
> 
>     }
> 
> }
> 
> ```

No offense intended, but I don't see a substantial difference. Unless this fixes something I was unaware of.

**Comment by rashil2000** [comment]
There's a lot of difference. Use the diff tool to see it.

**Comment by rashil2000** [comment]
There's no formatting difference. Please check all JSON fields one by one before commenting.

**Comment by AdmnJ** [comment]
> ```json
> {
> 
>     "version": "2.1.0",
> 
>     "description": "A Go library & CLI for undocumented Crunchyroll APIs.",
> 
>     "homepage": "https://github.com/ByteDream/crunchyroll-go",
> 
>     "license": "LGPL-3.0-or-later",
> 
>     "suggest": {
> 
>         "ffmpeg": "ffmpeg",
> 
>         "vcredist": "extras/vcredist2010"
> 
>     },
> 
>     "notes": "You need a premium account for full (API) access.",
> 
>     "architecture": {
> 
>         "64bit": {
> 
>             "url": "https://github.com/ByteDream/crunchyroll-go/releases/download/v2.1.0/crunchy-v2.1.0_windows.exe#/crunchy.exe",
> 
>             "hash": "76b746c16269981230c68bb4f85fc0a35ceaf68a124e9f882ee3584ea179d355"
> 
>         }
> 
>     },
> 
>     "bin": "crunchy.exe",
> 
>     "checkver": "github",
> 
>     "autoupdate": {
> 
>         "architecture": {
> 
>             "64bit": {
> 
>                 "url": "https://github.com/ByteDream/crunchyroll-go/releases/download/v$version/crunchy-v$version_windows.exe#/crunchy.exe"
> 
>             }
> 
>         }
> 
>     }
> 
> }
> ```

I did check all JSON fields before commenting, what I meant by formatting difference is that as my manifest worked in my testing so I am asking what was wrong with the original manifest that justifies the changes. I want to know the reasoning behind the changes, I can see them and I did not just glance over them. I want to know the reason behind each change as I want to know if I did something wrong. I am not inclined to change anything without knowing a reason behind the change. If it is just the preferred way to do it for the repo then please clarify that and the best practices(I read the guidelines but please tell me if I missed a requirement). I made the manifest based off of examples in the main and extra bucket and I am just wondering if your suggested edits are just how you would do it, or if it is how we are supposed to do it. Furthermore if it is just your preference, then why do you prefer it that way personally. 

It is the why to the changes, not the changes themselves...

**Comment by rashil2000** [comment]
There's nothing wrong with calling the bin multiple times, but why complicate things when they can be done in a simpler manner? You'll find lots of manifests where binaries are renamed in the URL itself. Also, only official binary shim names are allowed; you can't create custom-named shims.

As for description, the premium account requirement is not a "description" of the package per se, but rather a gotcha, or a "thing to keep in mind", which is exactly why the notes field exists.

**Comment by rashil2000** [comment]
Some other differences:
- The package is a compiled binary. There's no need for suggesting the Golang package. Moreover, Go binaries are fat binaries, so even vcredist is not required, but I left it there because some other (non-Go) packages have it.
- When the homepage is a GitHub repo, you can simplify the `checkver` to just say `github`, it will automatically use the repo from the homepage field.

**Comment by AdmnJ** [comment]
@rashil2000 Thanks for the clarifications, I have made a commit adopting most of them. I just wanted to know the reason behind them because I new there had to be a reason.

File: bucket/crunchyroll-go.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "2.1.0",
+    "description": "A Go library & cli for the undocumented crunchyroll api.",
```
Comment (rashil2000) [inline bucket/crunchyroll-go.json:3]:
```suggestion
    "description": "A Go library & CLI for the undocumented Crunchyroll API.",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]
No worries, I should've explained myself earlier.

Also, please remove any blank newlines.

**Comment by AdmnJ** [comment]
> No worries, I should've explained myself earlier.
> 
> Also, please remove any blank newlines.

I approved your commit and removed some blank lines 
Thumbs Up

File: bucket/crunchyroll-go.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "2.1.0",
+    "description": "A Go library & CLI for the undocumented Crunchyroll API.",
+    "homepage": "https://github.com/ByteDream/crunchyroll-go",
+    "license": "LGPL-3.0 License",
```
Comment (rashil2000) [inline bucket/crunchyroll-go.json:5]:
```suggestion
    "license": "LGPL-3.0-only",
```

License should be a valid SPDX identifier. https://spdx.org/licenses/

Comment (AdmnJ) [reply bucket/crunchyroll-go.json:5]: thought I made that change already, must have accidently removed it.

**Comment by rashil2000** [comment]
Thanks for your patience!

**Comment by AdmnJ** [comment]
> Thanks for your patience!

Thanks for the help and pointers.

**Pull request merged**